lib-injection/Dockerfile

```diff
@@ -9,22 +25,52 @@ RUN mkdir -p pkgs
 ADD ./dl_wheels.py .
 # Note that we only get Python >= 3.7. This is to keep the size of the image
 # as small as possible.
+RUN python3 dl_wheels.py \
+        --python-version=3.10 \
+        --platform="manylinux2014" \
+        --ddtrace-version=${DDTRACE_PYTHON_VERSION} \
+        --output-dir /build/pkgs \
+        --verbose
+
+FROM python:3.11-alpine
+WORKDIR /build
+ARG DDTRACE_PYTHON_VERSION
+RUN apk add gcc python3-dev musl-dev linux-headers
+RUN python3 -m pip install -U pip==23.0.1
+RUN python3 -m pip install packaging==23.0
+RUN mkdir -p pkgs
+ADD ./dl_wheels.py .
+# Note that we only get Python >= 3.7. This is to keep the size of the image
+# as small as possible.
 RUN python3 dl_wheels.py \
         --python-version=3.11 \
+        --platform="musllinux_1_1" \
+        --ddtrace-version=${DDTRACE_PYTHON_VERSION} \
+        --output-dir /build/pkgs \
+        --verbose
+
+FROM python:3.10-alpine
```

Comment:
can we merge this stage with the previous? Any reason to have to do it per python version?

Reply:
my conclusion was that we'd need multiple stages in order to build the wheels with `pip wheels` for each architecture and platform.